tests/nominal/test_MeanResponseTransformer.py

```diff
@@ -1141,77 +1049,6 @@ def expected_df_10():
 
         return df
 
-    def test_check_is_fitted_called(self, mocker):
-        """Test that BaseTransformer check_is_fitted called."""
-        df = create_MeanResponseTransformer_test_df()
-
-        x = MeanResponseTransformer(columns="b")
-
-        x.fit(df, df["a"])
-
-        expected_call_args = {0: {"args": (["mappings"],), "kwargs": {}}}
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.base.BaseTransformer,
-            "check_is_fitted",
-            expected_call_args,
-        ):
-            x.transform(df)
-
-    def test_not_dataframe_error_raised(self):
-        df = create_MeanResponseTransformer_test_df()
-
-        x = MeanResponseTransformer(columns="b")
-        x.fit(df, df["a"])
-
-        with pytest.raises(
-            TypeError,
-            match=f"{x.classname()}: X should be a pd.DataFrame",
-        ):
-            x.transform(X=[1, 2, 3, 4, 5, 6])
-
-    def test_super_transform_called(self, mocker):
-        """Test that BaseNominalTransformer.transform called."""
-        df = create_MeanResponseTransformer_test_df()
-
-        x = MeanResponseTransformer(columns="b")
-
-        x.fit(df, df["a"])
-
-        expected_call_args = {
-            0: {"args": (create_MeanResponseTransformer_test_df(),), "kwargs": {}},
-        }
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.nominal.BaseNominalTransformer,
-            "transform",
-            expected_call_args,
-            return_value=create_MeanResponseTransformer_test_df(),
-        ):
-            x.transform(df)
-
-    def test_learnt_values_not_modified(self):
```

Comment:
Q: how come we're dropping this one?

Reply:
that was a mistake.  added back in https://github.com/lvgig/tubular/pull/262/commits/2bcaab00452ea73a41d15efc5e75c299eea62ad1